cairo/src/utils.rs

```diff
@@ -47,6 +47,12 @@ impl Version {
     }
 }
 
+impl std::default::Default for Version {
```

Comment:
```suggestion
impl Default for Version {
```
I think the Default trait is in-scope by default?